core/src/apps/cardano/sign_tx.py

```diff
@@ -466,6 +464,14 @@ def _cborize_byron_witnesses(
 async def _show_standard_tx(
     ctx: wire.Context, keychain: seed.Keychain, msg: CardanoSignTx
 ) -> None:
+    is_network_id_verifiable = _is_network_id_verifiable(
```

Comment:
any reason for not passing the `CardanoSignTx` message instead of the individual parameters?

Reply:
no strong reason, it was a matter of my preference to pass what's really needed and make the interface more understandable in that way but I can see your point that this approach also results in exposing quite low-level concerns that may be fair to hide from the caller